tests/shared/src/test/scala/scala/meta/tests/tokenizers/TokenizerSuite.scala

```diff
@@ -1023,4 +1023,25 @@ class TokenizerSuite extends BaseTokenizerSuite {
     )
 
   }
+
+  test("Multiline interpolated string - ignore escape") {
+    val Tokens(
+      BOF(),
+      Interpolation.Id("raw"),
+      Interpolation.Start(),
+      Interpolation.Part("\\"),
+      Interpolation.SpliceStart(),
+      Ident("host"),
+      Interpolation.SpliceEnd(),
+      Interpolation.Part("\\"),
+      Interpolation.SpliceStart(),
+      Ident("share"),
+      Interpolation.SpliceEnd(),
+      Interpolation.Part("\\"),
+      Interpolation.End(),
+      EOF()
+    ) =
+      ("raw\"\"\"\\$host\\$share\\\"\"\"").tokenize.get
```

Comment:
do we need to handle the non-multiline `raw` strings: `raw"a\nb"`

https://docs.scala-lang.org/overviews/core/string-interpolation.html#the-raw-interpolator

Reply:
this sample works well